src/ChooseAGameName/GamePanel.java

```diff
@@ -103,7 +90,8 @@ public void compassPage(){
 	
 	public void actionPerformed(ActionEvent e) {
 		JButton buttonPressed = (JButton) e.getSource();
-		if (buttonPressed == mapButton) {
+		System.out.println(buttonPressed);
```

Comment:
Is this print statement still necessary?

Reply:
It t'was not ;)